src/frozen_fields/config.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "",
+  "properties": {
+    "hotkeyOne": {
+      "type": "string",
+      "title": "One field",
```

Comment:
Thanks! I think it would be best to make these a bit more descriptive, though, e.g.: "Hotkey for current field"...

Reply:
You want me to do the change and push again ?
By the way, why "hot key" and not "short cut" ? It's the first time I see this word